src/garage/np/algos/off_policy_rl_algorithm.py

```diff
@@ -72,6 +72,8 @@ def __init__(self,
 
         self.sampler_cls = OffPolicyVectorizedSampler
 
+        self.evaluation_batch = None
```

Comment:
any good reason this needs to be stored instead of just fetched and sent to evaluate_performance in a single block?